### PR TITLE
Add fingerprint

### PR DESCRIPTION
**Pull request by wangym5106**
Add fingerprint supported by libpg_query(https://github.com/lfittl/libpg_query#usage-fingerprinting-a-query)

### CODE REVIEW

File: pglast/__init__.py [hunk 1]
```diff
@@ -127,4 +127,4 @@ def _remove_stmt_len_and_location(parse_tree):
 
 
 __all__ = ('Error', 'Missing', 'Node', 'enums', 'get_postgresql_version',
-           'parse_plpgsql', 'parse_sql', 'prettify', 'split')
+           'parse_plpgsql', 'parse_sql', 'prettify', 'split', 'fingerprint')
```
Comment (lelit) [inline pglast/__init__.py:130]: Nitpicking hat: can you please keep the symbols in alphabetic order here and in the import statements?

**Review by lelit — COMMENTED** [review]
Having never used the `fingerprint` feature, I'd appreciate a mention of the new function in the doc, bonus point for a brief explanation of its purpose.

**Comment by lelit** [comment]
Thank you, will consider this as soon as possible.

File: pglast/__init__.py (hunk 1 above)
Comment (wangym5106) [reply pglast/__init__.py]: Fixed

**Comment by wangym5106** [comment]
Hi, I've added the description of the function to the docs.

**Comment by lelit** [comment]
I will try to integrate this over the next weekend.

**Pull request merged**